### PR TITLE
Added regex to CENSOR_CMDS to exclude sensitive commands used in api-server test cases.

### PR DESCRIPTION
**Pull request by jmekkatt**
@xingxingxia @wangke19 

- Added the regex pattern to CENSOR_CMDS (from /lib/local_process.rb) for excluding sensitive information shown in commands used in api-server test cases.
- In tests , We are calling _oc exec_ command and passing _skopeo_ command definition as an argument . _skopeo_ command contains the --dcreds option followed a username and password. 
See here http://pastebin.test.redhat.com/1013376
- With this PR , we would avoid printing during the test runs and make sure our sensitive info is not exposed.
- Test case execution result against the PR could find here : http://pastebin.test.redhat.com/1013382
- Reference ticket is OCPQE-7094.
Kindly review the change and let me know your feedback. Thanks

### CODE REVIEW

File: lib/local_process.rb [hunk 1]
```diff
@@ -11,13 +11,14 @@ class LocalProcess
...
     CENSOR_CMDS = [
-      'oc exec .*bearer',   # oc exec elasticsearch-* -- curl "Authorization: Bearer ***"
-      'oc rsh .*bearer',    # oc rsh elasticsearch-* -- curl "Authorization: Bearer ***"
-      'oc login .*--token', # oc login --token=***
-                            # oc login --token ***
-      'oc login .*-p',      # oc login --passwordr= ***
-                            # oc login --password ***
-                            # oc login -p ***
+      'oc exec .*bearer',    # oc exec elasticsearch-* -- curl "Authorization: Bearer ***"
+      'oc exec .*--dcreds',  # oc exec -- skopeo copy --dcreds username:****
```
Comment (liangxia) [inline lib/local_process.rb:15]: Searching the logs, i got,
```
skopeo --debug --insecure-policy copy --dest-tls-verify\=false --dcreds ***
skopeo inspect --tls-verify\=false --creds ***
```
Perhaps use `oc exec .*--d?creds` to cover both of them ?

Comment (wangke19) [reply lib/local_process.rb:15]: @jmekkatt Just like liangxia mentioned, we have other places where use '-creds', 

```
$ grep -nrE '\-\-\d?creds'
verification-tests/features/online/imagestreams.feature:289:      | --dcreds                                                    |
verification-tests/features/build/buildlogic.feature:525:      | --creds                                                                    |
verification-tests/features/build/buildlogic.feature:542:      | --creds                                           |
verification-tests/features/registry/remote.feature:29:      | --creds                    |
verification-tests/features/registry/remote.feature:39:      | --dcreds                   |
verification-tests/features/registry/remote.feature:57:      | --creds                    |
cucushift/online/imagestreams.feature:68:      | --dcreds                                                             |
cucushift/online/imagestreams.feature:97:      | --dcreds                                                                |
cucushift/online/imagestreams.feature:115:      | --dcreds                                                                |
cucushift/online/imagestreams.feature:144:      | --dcreds                                                                |
cucushift/build/buildlogic.feature:87:      | --creds            |
cucushift/build/buildlogic.feature:363:      | --creds            |
cucushift/build/buildlogic.feature:395:      | --creds            |
cucushift/cli/limits.feature:196:      | --dcreds                                  |
cucushift/admin/quota.feature:744:      | --dcreds                                                                 |
cucushift/registry/registry.feature:53:      | --dcreds                   |
cucushift/registry/remote.feature:26:      | --dcreds                             |
cucushift/registry/remote.feature:45:      | --dcreds                             |
cucushift/registry/remote.feature:159:      | --dcreds                             |

```

Comment (wangke19) [reply lib/local_process.rb:15]: one more tip, give the test results pipe output `| sed 's/\[[0-9;]*[mGKF]//g' | rhpaste` to remove ANSI-colors tags. looks better.

**Comment by jmekkatt** [comment]
@wangke19 @liangxia I have made the changes as per the suggestion. Could you please review it again?

**Pull request merged**